tests/conftest.py

```diff
@@ -320,6 +320,20 @@ def start_swss(self):
             cmd += "supervisorctl start {}; ".format(pname)
         self.runcmd(['sh', '-c', cmd])
 
+    #stop add daemons
+    def stop_all_daemons(self):
+        cmd = ""
+        for pname in self.alld:
+            cmd += "supervisorctl stop {}; ".format(pname)
+        self.runcmd(['sh', '-c', cmd])
+
+    #start all daemons
```

Comment:
I would rather remove this function since during the start, we probably need consider the order.   Stop path is OK as it will kill all daemons anyways, even though it may hit some timing issue and syslog errors.

Reply:
Agree